frontend/javascripts/oxalis/view/help_modal.tsx

```diff
@@ -0,0 +1,98 @@
+import { sendHelpEmail, updateNovelUserExperienceInfos } from "admin/admin_rest_api";
+import { Modal, Input, Alert } from "antd";
+import { setActiveUserAction } from "oxalis/model/actions/user_actions";
+import { OxalisState } from "oxalis/store";
+import React, { CSSProperties, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+function HelpButton() {
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  const dispatch = useDispatch();
+  const activeUser = useSelector((state: OxalisState) => state.activeUser);
+
+  const discardButton = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    // prevent the modal from also being shown
+    e.stopPropagation();
+
+    if (!activeUser) return;
+
+    const [newUserSync] = updateNovelUserExperienceInfos(activeUser, {
+      hasDiscardedHelpButton: true,
+    });
+    dispatch(setActiveUserAction(newUserSync));
+  };
+
+  if (!activeUser) return null;
+  if (activeUser && activeUser.novelUserExperienceInfos.hasDiscardedHelpButton) return null;
```

Comment:
```suggestion
  if (activeUser.novelUserExperienceInfos.hasDiscardedHelpButton) return null;
```

since you are already checking for `!activeUser`